### PR TITLE
fix some rules

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

**Review by XiyunZhao — APPROVED** [review]

**Review by XiyunZhao — APPROVED** [review]

File: lib/openshift/quota_limits.rb [hunk 1]
```diff
@@ -18,6 +19,10 @@ def storage_requests_raw
       @storage_requests
     end
 
+    def cpu_raw
```
Comment (pruan-rht) [inline lib/openshift/quota_limits.rb:22]: @yapei I think `cup` and `cpu_raw` can be consolidated into just `cpu`

Comment (yapei) [reply lib/openshift/quota_limits.rb:22]: got it, let me update

Comment (yapei) [reply lib/openshift/quota_limits.rb]: updated, could you help review again ? @pruan-rht

**Review by pruan-rht — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**